dll/cpl/desk/lang/de-DE.rc

```diff
@@ -138,18 +138,19 @@ END
 
 IDD_ADVANCED_GENERAL DIALOGEX 0, 0, 253, 204
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Allgemein"
+CAPTION "General"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Anzeige", -1, 7, 6, 237, 56, WS_GROUP
-    LTEXT "Schriftgröße", -1, 14, 20, 222, 9
-    COMBOBOX IDC_FONTSIZE_COMBO, 14, 30, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
-    GROUPBOX "Kompatibilität", -1, 7, 67, 237, 97
-    LTEXT "Einige Programme könnten fehlerhaft arbeiten, wenn nach Änderung der Farbeinstellungen nicht neu gestartet wird.\r\n\nNach Änderung der Farbeinstellungen", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&Computer vor dem Anwenden der Farbeinstellungen neu starten.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "Farbeinstellungen ohne Neustart übernehmen.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "Vor dem Anwenden der neuen Farbeinstellungen nachfragen.", IDC_ASKME_RB, 14, 146, 228, 10
+    GROUPBOX "Display:", -1, 7, 6, 237, 105, WS_GROUP
```

Comment:
It doesn't look like the dialog is so different that you need to undo all translations.